### PR TITLE
Added a port of less, groff and file, updated the man-db port

### PR DESCRIPTION
**Pull request by Dennisbonke**

### CODE REVIEW

File: bootstrap.yml [hunk 1]
```diff
@@ -2193,6 +2193,61 @@ packages:
...
+      - args:
+        - '@THIS_SOURCE_DIR@/configure'
+        - '--host=x86_64-managarm'
+        - '--prefix=/usr'
+        - '--with-doc=no'
+      - args: 'sed -i s/"_GL_MATH_CXX_REAL_FLOATING_DECL_1 (signbit)"//g @THIS_SOURCE_DIR@/lib/math.in.h'
+      - args: 'sed -i s/"_GL_MATH_CXX_REAL_FLOATING_DECL_2 (signbit)"//g @THIS_SOURCE_DIR@/lib/math.in.h'
+      - args: 'sed -i s/"#   undef signbit"//g @THIS_SOURCE_DIR@/lib/math.in.h'
+        environ:
+          PAGE: 'A4'
```
Comment (avdgrinten) [inline bootstrap.yml:2223]: What does this do?

Comment (Dennisbonke) [reply bootstrap.yml:2223]: It sets the default page size to A4

**Comment by Dennisbonke** [comment]
The latest commit also adds a working port of file, it requires the inclusion of the `pread()` PR in mlibc

**Comment by avdgrinten** [comment]
Looks good to me now. Thank you for the PR.

**Comment by Dennisbonke** [comment]
The last two commits have some bug fixes in some ports and adds bash startup files, these being `/etc/profile` `/etc/bash.bashrc` and `/root/.bashrc`

**Pull request merged**